### PR TITLE
chore: use explicit-module-boundary-types

### PR DESCRIPTION
**Pull request by Shinigami92**
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md

### CODE REVIEW

**Review by ST-DDT — APPROVED** [review]

**Comment by ST-DDT** [comment]
Partial duplicate of https://github.com/faker-js/faker/pull/358

**Review by prisis — DISMISSED** [review]

**Comment by ST-DDT** [comment]
This needs to be rebased.

**Review by ST-DDT — APPROVED** [review]

**Review by import-brain — APPROVED** [review]

File: src/lorem.ts [hunk 1]
```diff
@@ -152,7 +152,16 @@ export class Lorem {
...
     return this.faker.fake('{{' + randomLoremMethod + '}}');
-  };
+  }
+
+  /**
+   * Alias for `text`.
+   *
+   * @param times
+   */
+  loremText(times?: number): string {
```
Comment (griest024) [inline src/lorem.ts:162]: Changing the public api should not be in chore PRs

**Review by griest024 — CHANGES_REQUESTED** [review]

File: src/lorem.ts (hunk 1 above)
Comment (Shinigami92) [reply src/lorem.ts:162]: I did not, this is an alias

Comment (griest024) [reply src/lorem.ts:162]: this adds a public function `faker.lorem.loremText` where it did not exist previously, correct?

Comment (Shinigami92) [reply src/lorem.ts:162]: no, see https://github.com/faker-js/faker/pull/361/files#diff-5b65c037dc3f0bd1a1693a84ed1514a704af5382782d4556efc12729759e25d3L143
Previously it had a `text = function loremText(times?: number) {`, I splitted this now, caue the `this.faker.fake` call was trapped by the function scope

**Review by griest024 — APPROVED** [review]

**Review by import-brain — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**